contracts/Curve.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity >=0.6.0 <0.7.0;
+
+contract Curve {
+  constructor() public {}
+
+  function price(uint256 supply, uint256 units)
+    external
+    view
+    virtual
+    returns (uint256) {
```

Comment:
This should not have an implementation (the `{}`). instead it should just have the function signature ending with a `;` I'm pretty sure